src/App.tsx

```diff
@@ -4,15 +4,15 @@ import logo from "./logo.svg";
 import "./App.css";
 import Voting from "./pages/voting";
 import Leaderboard from "./pages/leaderboard";
-
-//Test Comment
+import Header from "./pages/header";
 
 export const App = () => {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<Voting />} />
         <Route path="leaderboard" element={<Leaderboard />} />
+        <Route path="header" element={<Header />} />
```

Comment:
Routes allow you to render a component on a route like - /header. Notice the slash. The above line is wrong, it should be "/leaderboard".

In this case, Header doesn't need to be added to the routes. We don't really want the header to be rendered at "/header" so we can remove this.